js/common.js

```diff
@@ -0,0 +1,18 @@
+const themeToggleButton = document.getElementById("theme-toggle");
+
+const changeBackgroundColor = () => {
+  // Toggle the 'Dark' class on the <body>
+  document.body.classList.toggle('Dark');
```

Comment:
In here please use styling via css classes and not js (this way you don`t need all of your logic)